scripts/runtestsuite/runtestsuite.py

```diff
@@ -477,11 +495,15 @@ def render_test_scenes(script_directory, args):
                 if render_test_scene(args, logger, report_writer, dirpath, filename):
                     passing_scene_count += 1
 
-    report_writer.close()
+    end_time = datetime.datetime.now()
+    total_time = end_time - start_time
+    success = 100.0 * passing_scene_count / rendered_scene_count if rendered_scene_count > 0 else 0.0
+    failures = [rendered_scene_count - passing_scene_count, rendered_scene_count]
+    report_writer.close(total_time, success, failures)
```

Comment:
```suggestion
    end_time = datetime.datetime.now()
    total_time = end_time - start_time
    
    success = 100.0 * passing_scene_count / rendered_scene_count if rendered_scene_count > 0 else 0.0   
    failures = [rendered_scene_count - passing_scene_count, rendered_scene_count]
    
    report_writer.close(total_time, success, failures)
```

To make the core more readable, you can group them by "feature"